docs/prestodb-guide.md

```diff
@@ -301,3 +303,21 @@ Presto carbon only supports reading the carbon table which is written by spark c
 During reading, it supports the non-distributed indexes like block index and bloom index.
 It doesn't support Materialized View as it needs query plan to be changed and presto does not allow it.
 Also, Presto carbon supports streaming segment read from streaming table created by spark.
+
+## Presto Setup with CarbonData Distributed IndexServer
```

Comment:
As `prestosql` is default profile, add this doc in `prestosql-guide.md` and in prestodb doc give a link to this section of prestosql as it is common to both the presto version